CMakeLists.txt

```
@@ -4,6 +4,11 @@ project(OpenXRay)
 
 set(CMAKE_CXX_STANDARD 17)
 
+set(CMAKE_VERBOSE_MAKEFILE ON)
+if (CMAKE_VERBOSE_MAKEFILE)
+	set(CMAKE_EXECUTE_PROCESS_COMMAND_ECHO STDOUT)
```

Should use spaces instead of tabs.
```suggestion
    set(CMAKE_EXECUTE_PROCESS_COMMAND_ECHO STDOUT)
```